tools/pip/setup.py

```diff
@@ -148,20 +153,46 @@ def has_ext_modules(self):
 if variant.endswith('MKL'):
     if platform.system() == 'Darwin':
         shutil.copytree(os.path.join(CURRENT_DIR, 'mxnet-build/3rdparty/mkldnn/build/install/include'),
-                    os.path.join(CURRENT_DIR, 'mxnet/include/mkldnn'))
+                        os.path.join(CURRENT_DIR, 'mxnet/include/mkldnn'))
+
 if platform.system() == 'Linux':
     shutil.copy(os.path.join(os.path.dirname(LIB_PATH[0]), 'libgfortran.so.3'), os.path.join(CURRENT_DIR, 'mxnet'))
     package_data['mxnet'].append('mxnet/libgfortran.so.3')
     shutil.copy(os.path.join(os.path.dirname(LIB_PATH[0]), 'libquadmath.so.0'), os.path.join(CURRENT_DIR, 'mxnet'))
     package_data['mxnet'].append('mxnet/libquadmath.so.0')
+elif platform.system() == 'Windows':
+    package_data['mxnet'].append(os.path.abspath(os.path.join('mxnet', os.path.basename(LIB_PATH[0]))))
+    package_data['mxnet'].append(os.path.abspath(os.path.join('mxnet', 'libmxnet.lib')))
+    for root, dirs, files in os.walk('win_third/openblas'):
+        for file in files:
+            shutil.copy(os.path.join(root, file), os.path.join(CURRENT_DIR, 'mxnet'))
+            package_data['mxnet'].append(os.path.abspath(os.path.join('mxnet', file)))
+    if variant != 'CPU':
```

Comment:
could we wrap this in a function for readability and maint.?

Reply:
I actually don't think wrap with a function will improve readability in this case.

The code added is party of if...elif.. condition, move it into function make it harder to compare difference between Linux and Windows.